pkg/importer/dataStream.go

```diff
@@ -183,6 +212,38 @@ func (d dataStream) http() (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
+func (d dataStream) glance() (io.ReadCloser, error) {
+    opts := gophercloud.AuthOptions{
+        IdentityEndpoint: d.provider.IdentityEndpoint,
+        Username: d.accessKeyId,
+        Password: d.secretKey,
+        DomainName: d.provider.Domain,
+      }
+	  glog.V(Vuser).Infof("opts: %q", opts) 
+
+    provider, err := openstack.AuthenticatedClient(opts)
+	if err != nil {
+		glog.V(Vuser).Infof("AuthenticatedClient: %q", err)
+		panic(err)
+	}
+    client, err := openstack.NewIdentityV3(provider, gophercloud.EndpointOpts{})
+	if err != nil {
+		glog.V(Vuser).Infof("NewIdentityV3: %q", err)
```

Comment:
The log calls here and below don't make sense to me. They appear to be informational and then we panic. I expected something more like `glog.Errorf(...)`